modules/plugin/src/main/scala/org/scalasteward/plugin/StewardPlugin.scala

```diff
@@ -49,11 +51,19 @@ object StewardPlugin extends AutoPlugin {
         )
       val dependencies = libraryDeps ++ scalafixDeps
 
+      def getCredentials(url: URL): Option[Resolver.Credentials] =
+        Credentials
+          .forHost(sbtCredentials, url.getHost)
```

Comment:
[`forHost`](https://github.com/sbt/librarymanagement/blob/879bd5399c6d53a3c692513cd1e91d24c89eacc7/ivy/src/main/scala/sbt/librarymanagement/ivy/Credentials.scala#L36) throws if it can't load `FileCredentials`. If we let `stewardDependencies` return with an error in this case, Scala Steward won't be able to do anything in such a repo even if the credentials are only required for publishing. I think we should therefore return `None` here if `forHost` throws.

Reply:
True! Thanks for catching this!